v2.0/src/Components/Pages/ActionsPage/PhotoSensitiveAction.js

```diff
@@ -153,105 +150,97 @@ class PhotoSensitiveAction extends React.Component {
 
   checkDone() {
     var action = this.props.action;
-    // var households = this.props.user.households || [];
     var done = this.props.done ? this.props.done : [];
     var exists =
       done.filter((t) => t.action.id === action.id).length > 0 ? true : false;
     return exists;
   }
 
-  newRender() {
+  newFlexRender() {
     const actionStateCase = this.getActionStateCase();
     return (
       <div
         className={`col-lg-6 col-md-12 col-sm-12 col-12 ${MEAnimation.getAnimationClass()}`}
-        style={{ padding: 10 }}
+        key={this.props.key?.toString()}
```

Comment:
What does the question mark do in this? 
Just trying to understand javascript better